assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
-                "26L": ["_R8R-1DME", "OHOFO", "BORLE"],
-                "26R": ["_R8L-1DME", "OHOFO", "BORLE"],
-                "30" : ["_R12-1DME", "OHOFO", "BORLE"],
-                "27" : ["_R9-1DME", "OHOFO", "BORLE"]
+                "KMIA8L": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA8R": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA9": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA12": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA26L": ["OHOFO"],
+                "KMIA26R": ["OHOFO"],
+                "KMIA27": ["OHOFO"],
+                "KMIA30": ["OHOFO"]
             },
+            "body": ["BORLE", "MEAGN"],
             "exitPoints": {
-                "WINCO": ["MEAGN", "WINCO"]
+                "WINCO": ["WINCO"]
             },
-            "draw": [["FORKK", "BORLE"], ["OHOFO", "BORLE", "MEAGN", "WINCO*"]]
+            "draw": [
+                ["SENOY", "CIMBA", "FORKK", "BORLE", "MEAGN", "WINCO*"],
+                ["OHOFO", "BORLE"]
+            ]
         }
     },
     "stars": {
         "ANNEY3": {
             "icao": "ANNEY3",
             "name": "Anney Three",
             "entryPoints": {
-                "OMN": ["OMN", "PCMAN", "MLB", "TRV", ["PHORD", "A240"], "PBI", ["ANNEY", "A130|S280"]],
-                "ZFP": ["ZFP", "SNSBK", ["ANNEY", "A130|S280"]]
+                "MLB": ["MLB", "TRV", "PHORD", "PBI"],
+                "OMN": ["OMN", "PCMAN", "MLB", "TRV", "PHORD", "PBI"],
+                "SNSBK": ["SNSBK"],
+                "TRV": ["TRV", "PHORD", "PBI"]
+            },
+            "body": ["ANNEY", "HILEY", "KAINS", "VKZ"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["OMN*", "PCMAN", "MLB*", "TRV*", "PHORD", "PBI", "ANNEY", "HILEY", "KAINS", "VKZ"],
+                ["SNSBK*", "ANNEY"]
+            ]
+        },
+        "BLUFI4": {
+            "icao": "BLUFI4",
+            "name": "Bluffy Four",
+            "entryPoints": {
+                "MLB": []
+            },
+            "body": ["MLB", "LEBUR", "DALOO", "TOPPR", "BLUFI", "HEATT", "KAINS", "VKZ"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
             },
-            "body": ["HILEY", "KAINS", "VKZ"],
-            "draw": [["OMN*","PCMAN","MLB","TRV","PHORD","PBI","ANNEY","HILEY"], ["ZFP*","SNSBK"], ["HILEY","KAINS","VKZ"]]
+            "draw": [
+                ["MLB*", "LEBUR", "DALOO", "TOPPR", "BLUFI", "HEATT", "KAINS", "VKZ"]
+            ]
         },
         "CURSO4": {
```

Comment:
CURSO4 is now CURSO5

Reply:
reverified and updated all SIDs/STARs.

done